src/cli/error.rs

```diff
@@ -36,6 +37,9 @@ impl PastelError {
             PastelError::GradientNumberMustBeLargerThanOne => {
                 "The number of colors must be larger than one".into()
             }
+            PastelError::GradientColorCountMustBeLargerThanOne => {
+                "The number of colors must be larger than one".into()
```

Comment:
Maybe: "the number of color arguments must be larger than one"? And change the one above to: "the specified color count must be larger than one".